kmk/internal_state.py

```diff
@@ -29,20 +140,20 @@ def __init__(self, config):
 
     def __repr__(self):
         return (
-            'InternalState('
-            'keys_pressed={} '
-            'coord_keys_pressed={} '
-            'leader_pending={} '
-            'leader_last_len={} '
-            'hid_pending={} '
-            'leader_mode_history={} '
-            'active_layers={} '
-            'start_time={} '
-            'timeouts={} '
-            'tapping={} '
-            'tap_dance_counts={} '
-            'tap_side_effects={}'
-            ')'
+            "InternalState("
```

Comment:
This will fail the linter (actually, basically all strings in this PR seem to) because [we use single quotes](https://github.com/KMKfw/kmk_firmware/blob/master/setup.cfg#L6) as our lone deviation from the [Black style guide](https://github.com/psf/black)

There's maybe some argument to be made to stop deviating from Black's decisions, since Black, like `go fmt`, is meant to be a "one true way" solution, but the current status quo dates back to 2018 when I have more time to care about silly things like quoting style ¯\\_(ツ)_/¯

Reply:
`make black` should also just fix these sorts of things IIRC.